torchbenchmark/models/Background_Matting/__init__.py

```diff
@@ -51,8 +51,11 @@ def __init__(self, device=None, jit=False):
             'reso': (self.opt.resolution, self.opt.resolution)}
         traindata = VideoData(csv_file=csv_file,
                               data_config=data_config_train, transform=None)
-        self.train_loader = torch.utils.data.DataLoader(
+        train_loader = torch.utils.data.DataLoader(
             traindata, batch_size=self.opt.batch_size, shuffle=True, num_workers=self.opt.batch_size, collate_fn=_collate_filter_none)
+        self.train_data = []
+        for data in train_loader:
+            self.train_data.append(data)
```

Comment:
do you need to do `.to(device)` here?

Reply:
Good point! Originally we copy data to cuda in `train()`. We should do that in `__init__()` as well.

Reply:
Removed the redundant `cuda()` calls in `train()` because they are already copied to device in `__init__()`.